src/main/java/net/sf/jabref/gui/FileListEntry.java

```diff
@@ -57,6 +57,11 @@ public void setType(ExternalFileType type) {
         this.type = type;
     }
 
+    public String[] getStringArrayRepresentation() {
+        String type = getType() != null ? getType().getName() : "";
+        return new String[] {getDescription(), getLink(), type};
```

Comment:
I was thinking about that as well. ArrayList<ArrayList<String>>? But then I wasn't sure which was the most suitable one.

This especially holds for the encode and decode functions in StringUtil.